### ISSUE TITLE
Script to generate a default yaml

### ISSUE THREAD

**Issue by apeck12** [lcls-users/btx#286]
Not exactly related to the page _per se_ but maybe it is worth writing a script which will generate an example with sensible default values. So something like this from the command line:

```
$> generate_yaml <exp_name> <det_type>
```

which outputs the yaml in a hierarchy like this with prepopulated setup section and basic tasks:
```
${SIT_PSDM_DATA}/{instrument}/{exp}/scratch/btx/
└──yaml/
    └── default_config.yaml
```

_Originally posted by @gadorlhiac in https://github.com/lcls-users/lcls-users/pull/16#discussion_r1129587972_

And then the Introduction to btx page in lcls-users Read the Docs should be updated.

**Comment by fredericpoitevin** [comment]
See #210 for the discussion with Yilai

**Comment by fredericpoitevin** [comment]
The above `submit_main.py` is for cryoHub, here we are talking about some default for users.

**Comment by gadorlhiac** [comment]
Partially resolved by #339 

This doesn't adjust default values, but sets up a directory with a copy of a maintained "default" config, and adjusts things like the root directory. It also creates submission jobs.
